build.assets/Dockerfile-centos7

```diff
@@ -207,8 +249,7 @@ USER ci
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal --default-toolchain $RUST_VERSION && \
     rustup --version && \
     cargo --version && \
-    rustc --version && \
-    rustup component add --toolchain $RUST_VERSION-x86_64-unknown-linux-gnu rustfmt clippy
```

Comment:
Is this no longer needed?

Reply:
It was never needed. We use clippy and rustfmt in CI (ubuntu box), and this is our "release image". We do not run any of those during release, and we really shouldn't.